concourse/pipelines/guest-package-build.yaml

```diff
@@ -1596,6 +1596,398 @@ jobs:
       result_state: "failure"
       start_timestamp: ((.:start-timestamp-ms))
 
+- name: build-compute-image-windows
+  plan:
+  - get: compute-image-windows
+    trigger: true
+    params:
+      skip_download: true
+  - load_var: commit-sha
+    file: compute-image-windows/.git/ref
+  - get: guest-test-infra
+  - task: generate-timestamp
+    file: guest-test-infra/concourse/tasks/generate-timestamp.yaml
+  - load_var: start-timestamp-ms
+    file: timestamp/timestamp-ms
+  - task: get-github-token
+    file: guest-test-infra/concourse/tasks/get-github-token.yaml
+  - task: generate-timestamp
+    file: guest-test-infra/concourse/tasks/generate-timestamp.yaml
+  - load_var: timestamp
+    file: timestamp/timestamp
+  - task: compute-image-windows-integration-test
+    file: guest-test-infra/concourse/tasks/package-gointegtest.yaml
+    vars:
+      repo-name: compute-image-windows
+      timestamp: ((.:timestamp))
+  - task: validate-integtest-result
+    file: guest-test-infra/concourse/tasks/validate-integtest-result.yaml
+    vars:
+      timestamp: ((.:timestamp))
+      repo-name: compute-image-windows
```

Comment:
have you built support for windows into this task? it runs a daisy workflow that creates a linux vm, checks out the code, and runs go test -tags integ on it.

Reply:
Removing the validate-integtest-result for compute-image-windows build.